app/lib/katello/resources/cdn/katello_cdn.rb

```diff
@@ -33,8 +33,8 @@ def fetch_paths(content_path)
           results = json_body['results']
 
           results.map do |repo|
-            Katello::Content.substitute_content_path(arch: repo[:arch],
-                                                     releasever: repo[:minor],
+            Katello::Content.substitute_content_path(arch: repo['arch'],
+                                                     releasever: repo['minor'],
                                                      content_path: content_path)
```

Comment:
Can we do something like:
repo_hash = HashWithIndifferentAccess.new(repo) and then we won't have to worry about symbol or string key access?
Thoughts?

Reply:
I initially wanted this approach, but decided against it because I wasn't sure about performance characteristics in case `results` or `repo` is particularly large.

If you think it's fine, I'll happily make the change.

Reply:
I like @wbclark 's approach better. Only thing I 'd suggest adding is a Unit test.